diff --git a/addon/components/render-template.js b/addon/components/render-template.js
index 72e2450a2fff51323fc982e62e605f1f57ab0668..12b40402957f8816a46c4cdcb546914b3c947aa3 100644
--- a/addon/components/render-template.js
+++ b/addon/components/render-template.js
@@ -2,7 +2,6 @@ import Component from '@ember/component';
 import { setOwner, getOwner } from '@ember/application';
 import { once } from '@ember/runloop';
 import { compileTemplate } from '@ember/template-compilation';
-import { assign } from '@ember/polyfills';
 import layout from '../templates/components/render-template';
 
 
@@ -20,7 +19,7 @@ export default Component.extend({
       let _props = this.get('props') || {};
       let domForAppWithGlimmer2 = owner.lookup('service:-document');
 
-      let props = assign({}, _props, {
+      let props = Object.assign({}, _props, {
         layout: compileTemplate(this.get('templateString') || ''),
       });
 